go/go.mod

```diff
@@ -57,8 +57,8 @@ require (
 	github.com/uber/jaeger-client-go v2.16.0+incompatible
 	github.com/uber/jaeger-lib v2.0.0+incompatible // indirect
 	github.com/whyrusleeping/go-logging v0.0.1
-	github.com/zondax/ledger-go v0.11.0
-	github.com/zondax/ledger-oasis-go v0.3.0
+	github.com/zondax/ledger-go v0.12.0
```

Comment:
Can you single out these package updates (go.mod and go.sum changes) to a separate commit?

Reply:
They were in a different commit and I was just asked to squash everything...